packages/@contentful--app-scripts/lib/create-app-definition/get-management-token.js

```diff
@@ -17,6 +21,7 @@ async function getManagementToken() {
     console.log(err.message);
     throw err;
   }
+  const cachedAccessToken = process.env[ACCESS_TOKEN_ENV_KEY];
 
   const { mgmtToken } = await inquirer.prompt([
```

Comment:
Don't we want to skip the prompt when there is a token in the cache?

Reply:
From the discussion and notes, we want to use it as a fallback. @fabe wdyt?

Reply:
I see, it's just that I had the feeling as this is opening the browser to copy the token I'm not sure if it's clear that you can skip that step to use the cached one, but it's just my personal feeling

Reply:
Ideally, if there's a token cached we just use that without the prompt. If it fails we ask for it again. I guess that's meant with "fallback" as well?

Reply:
I'd also just use it if we've got it stored and retrigger the flow if it's invalid.

If we still ask but use the stored value as the default, then how do we indicate that the default is there? We should never output the value (input is "password", but how does it behave with defaults?)

Reply:
Whenever we cache things - we log loud and clear that we did so. For app definitions & org, we auto-select in interactive mode (user only has to hit enter). It works the same with access token but we dont log anything so, user has to be aware of this behaviour. We can start with cache data as @loweisz & @fabe suggested and resort to original flow if not.